tests/unit/parts/test_update_metadata.py

```diff
@@ -296,14 +295,54 @@ def test_update_project_metadata_multiple(
     assert project.description == expected["description"]
     assert project.title == expected["title"]
     assert project.grade == expected["grade"]
-    assert project.contact == ["test@test.com"]
+    assert project.contact == ["contact1", "contact2"]
     assert project.license == "GPL-3.0"
     assert project.donation == ["donation1", "donation2"]
     assert project.source_code == ["source-code", "vcs-browser"]
     assert project.issues == ["issues1", "issues3", "issues2"]
     assert project.website == ["website1", "website2"]
 
 
+def test_update_project_metadata_appstream(new_dir):
```

Comment:
Which are you trying to test here:
1. `snapcraft.yaml` metadata overrides the appstream?
2. invalid metadata links are ignored?

Reply:
The overriding of metadata. Test of malformed links can't be done here. Should I change the name?

Reply:
Yeah, I think you should change the name or add a docstring to the test.


 Also, if you use valid URLs/emails in the appstream metadata here, then we will know they are overridden and not ignored for being invalid.

Reply:
Done